plugins/ScheduledReports/templates/_listReports.twig

```diff
@@ -75,12 +75,19 @@
             </td>
             <td>
                 {# download link #}
-                <a href="{{ linkTo({'module':'API', 'segment': null, 'token_auth':token_auth,
-                            'method':'ScheduledReports.generateReport', 'idReport':report.idreport,
-                            'outputType':downloadOutputType, 'language':language,
-                            'format': (report.format in ['html', 'csv']) ? report.format : false
-                       }) }}"
-                   rel="noreferrer noopener" target="_blank" name="linkDownloadReport" id="{{ report.idreport }}" class="link_but withIcon">
+                <form action="{{ linkTo({ 'module':'API', 'segment': null,
+                    'method':'ScheduledReports.generateReport', 'idReport':report.idreport,
+                    'outputType':downloadOutputType, 'language':language,
+                    'format': (report.format in ['html', 'csv']) ? report.format : false }) }}"
+                      method="POST"
+                      target="_blank"
+                      id="downloadReportForm_{{ report.idreport }}"
```

Comment:
haven't tested it yet but code looks good overall. It is not really needed since we know the id will be a number but just to be safe in HTML attributes we escape the output of twig variables using `html_attr` like this:

```
{{ report.idreport|e('html_attr') }}
```

instead of the standard escaping functionality (the standard escaping isn't safe for attributes). Not really needed when we know it's a number, but good to have eg if someone else copy/pastes it or changes it from an ID to something else etc.